src/modules/adsupport.js

```diff
@@ -464,6 +464,7 @@ export default function (playerInstance, options) {
             duration = duration || playerInstance.vastOptions.duration;
 
             playerInstance.nonLinearTracking = setInterval(function () {
+                console.log('nonLonearTracking')
```

Comment:
Don't know, but is this meant to be here or did you forget to remove it?

Reply:
I forgot to remove it, nice catch!